src/components/NewMessage/NewMessage.vue

```diff
@@ -382,11 +421,11 @@ export default {
 		},
 
 		showAttachmentsMenu() {
-			return this.canShareFiles && !this.broadcast && !this.upload
+			return this.canShareFiles && !this.broadcast && !this.upload && !this.messageToEdit
```

Comment:
It would be good to just disable button, when to hide it. less jumping on the screen

`<NcActions :disabled="this.messageToEdit" />` should do the trick